tests/unit_tests/pyanaconda_tests/modules/storage/test_module_device_tree.py

```diff
@@ -849,6 +849,26 @@ def test_set_device_mount_options(self):
         self.interface.SetDeviceMountOptions("dev1", "")
         assert dev1.format.options == "defaults"
 
+    def test_get_required_mount_points(self):
+        """Test GetRequiredMountPoints."""
+        result = self.interface.GetRequiredMountPoints()
+        assert isinstance(result, list)
+        assert len(result) != 0
+
+        for mp in result:
+            assert mp["mount-point"] is not None
+            assert mp["required-filesystem-type"] is not None
```

Comment:
You are testing directly the type transported on D-Bus, while it's possible to convert back to the structure list. I am not 100% sure which is correct when, but I'd expect the former as it's less "raw".

Here's a few lines to show how that would look like:
```
result = RequiredMountPoint.from_structure_list(self.interface.GetRequiredMountPoints())
for mp in result:
    assert mp.mount_point is not None
root = next(r for r in result if r.mount_point == "/")
```

Your thing works, just... it's writing the python test as if it was the javascript frontend.

Reply:
Ok, fixed.